### ISSUE TITLE
Accessibility issue for fd-icons : The icons are focusable but lack a role attribute 

### ISSUE THREAD

**Issue by MonishaK28** [SAP/fundamental-ngx#12805]
In https://sap.github.io/fundamental-ngx/#/core/icon, the icons are focusable but they do not have any role attribute defined due to which it is not read out during reading mode.  

![image](https://github.com/user-attachments/assets/cb1097d4-7b81-4037-bc62-adc693b4fa1e)

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)
Latest

#### Is there anything else we should know?

**_IMPORTANT: Please refrain from providing links or screenshots of SAP's internal information, as this project is open-source, and its contents are accessible to anyone._**

**Comment by droshev** [comment]
@MonishaK28 this is just to show the icons available in the font, it is not meant to be used for any actions.
